deployment/docker-compose.yml

```diff
@@ -52,6 +52,7 @@ services:
       MERLIN_WORKER_DB_SERVER: postgres
       MERLIN_WORKER_DB_USER: "${AERIE_USERNAME}"
       MERLIN_WORKER_LOCAL_STORE: /usr/src/app/merlin_file_store
+      SIMULATION_EXTENT_MINIMUM_POLL_PERIOD_MILLIS: 2000
```

Comment:
Can we add this new env var to the environment var document? https://github.com/NASA-AMMOS/aerie/blob/develop/deployment/Environment.md

Reply:
Absolutely: https://github.com/NASA-AMMOS/aerie/blob/d86e6b084707b1e7ae2e53384c1d5ce5cf0b8658/deployment/Environment.md